src/data/terraform.json

```diff
@@ -126,7 +126,7 @@
 		},
 		{
 			"iconName": "docs",
-			"name": "Plugin sdkv2",
+			"name": "SDKv2",
```

Comment:
Minor text updates here, figured I'd knock em out since they're very closely related. 🎟️ [[terraform-plugins-sdkv2] fix breadcrumbs project name & version](https://app.asana.com/0/1202097197789424/1202785022290147/f)